recipes/osx-sysroot/conda_build_config.yaml

```diff
@@ -0,0 +1,16 @@
+sdk_version:
+  - "10.9"
+  - "10.10"
+  - "10.11"
+  - "10.12"
+  - "10.13"
+  - "10.14"
+  - "10.15"
+  - "11.0"
+  - "11.1"
+  - "11.3"
+
+arm_sdk_version:
+  - "11.0"
+  - "11.1"
+  - "11.3"
```

Comment:
Why not make a zip between `version` and `cross_target_platform`? That way we could also simplify the recipe to one output.